src/pages/orders/withOrderController.tsx

```diff
@@ -8,28 +8,31 @@ import { useObservable } from 'stores/useObservable';
 import { UsersService } from 'stores/user';
 import { EVENT_NAME, EventAction } from 'utils';
 
-type GroupOrders = { [groupId: string]: Order[] };
+type IGroupOrders = { [groupId: string]: Order[] };
```

Comment:
@phanvohieunghia  This is a type, not an interface.
Why add the letter I?
Should interface names begin with an "I" prefix?

I think using "I" as a prefix is unnecessary because the choice depends on personal preference or project rules.